org.eclipse.jdt.ui.tests.refactoring/test cases/org/eclipse/jdt/ui/tests/refactoring/ExtractInterfaceTests.java

```diff
@@ -135,6 +135,40 @@ protected void validatePassingTest(String className, String[] cuNames, String ne
 		assertEqualLines("(interface cu)", getFileContents(getOutputTestFileName(newInterfaceName)), interfaceCu.getSource());
 	}
 
+	protected void validatePassingTest(String className, String[] cuNames, String newInterfaceName, boolean replaceOccurrences, String[] extractedMethodNames, String[][] extractedSignatures, String[] extractedFieldNames, IPackageFragment newPackageFragment) throws Exception {
```

Comment:
I think it would be a good idea to range format this method, since there is some odd formatting (mixed tabs/spaces, long lines).